website/source/docs/operating-a-job/update-strategies/index.html.md

```diff
@@ -12,10 +12,10 @@ description: |-
 
 Most applications are long-lived and require updates over time. Whether you are
 deploying a new version of your web application or upgrading to a new version of
-redis, Nomad has built-in support for rolling updates. When a job specifies a
-rolling update, Nomad can take some configurable strategies to minimize or
-eliminate down time, stagger deployments, and more. This section and subsections
-will explore how to do so safely with Nomad.
+redis, Nomad has built-in support for rolling, blue/green, and canary updates.
```

Comment:
nitpick: Redis or `redis`

(Either capitalize the name of the project, or if it's meant to refer to the image name enclose it in ticks)